src/components/shared/AccountNumberField.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+	<input
+		id="iban"
+		class="input"
+		:placeholder="placeholder"
+		:value="displayValue"
+		@blur="$emit( 'validate' )"
+		@input="onInput"
+		@paste="onInput"
+	/>
+</template>
+
+<script setup lang="ts">
+
+import { ref, watch } from 'vue';
+
+interface Props {
+	id?: string;
+	placeholder: string;
+	accountId: string;
+	spaceIndexes?: number;
```

Comment:
I was confused by this name, maybe name it very literally `insertSpaceEveryNCharacter`. Or `characterGroupSize`. Or `displayGroupSize`. Or `numCharactersPerDisplayGroup`. Or something else, that expresses the domain better